tools/benchmark/cmd/root.go

```diff
@@ -48,10 +46,3 @@ func init() {
 	RootCmd.PersistentFlags().UintVar(&totalConns, "conns", 1, "Total number of gRPC connections")
 	RootCmd.PersistentFlags().UintVar(&totalClients, "clients", 1, "Total number of gRPC clients")
 }
-
-func Execute() {
```

Comment:
Or we should change the main func to use this func? Change cmd.RootCmd.Execute() to cmd.Execute()?